### ISSUE TITLE
system table compaction_history not updated

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla#683]
Even after running nodetool compact, the history table is empty
```
cqlsh> select * from system.compaction_history;

 id | bytes_in | bytes_out | columnfamily_name | compacted_at | keyspace_name | rows_merged
----+----------+-----------+-------------------+--------------+---------------+-------------

(0 rows)
```
The log do shows compacts was executed
```
Dec 22 10:56:39 tzach-x240 scylla[13726]:  [shard 2] compaction - Compacted 2 sstables to [/var/lib/scylla/data/system/schema_usertypes-3aa752254f82350b8d5c430fa221fa0a/system-schema_usertypes-ka-30-Data.db:level=0, ]. 44 bytes to 22 (~50% of original) in 89ms = 0.000234985MB/s. ~512 total partitions merged to 1.
```

**Comment by tzach** [comment]
tested with scylla 0.13.2 rpm
scylla-server-debuginfo-0.13.2-20151216.43d3c5f.fc22.x86_64.rpm

**Comment by slivne** [comment]
this was merged past 0.13.2 a2eac711cf0f383530523da707f14854d2a94136

**Comment by slivne** [comment]
related to #236
